src/components/trace/hat-script-graph.ts

```diff
@@ -143,7 +143,7 @@ class HatScriptGraph extends LitElement {
     const trace = this.trace.trace[path] as ChooseActionTraceStep[] | undefined;
     const trace_path = trace?.[0].result
       ? trace[0].result.choice === "default"
-        ? [config.choose.length]
+        ? [config.choose?.length || 0]
```

Comment:
We should update the type that it can be `null` too. Or should we update the backend to make it always an empty list? 

Actually, from the backend:

```python
vol.Required(CONF_CHOOSE): vol.All(
```

So how can it be `null` ? 🤔

Reply:
I think that we should fix this issue in the backend.

Reply:
oh nevermind, this is the pre-validated config. Yes it can be `null`, it will become an empty list.

Reply:
So this is good?